Added :

jacklundy96 opened the pull request:
.Fragments have been refractored to be safer too use and tidier code via a method for each fragment
.Emplyment stats page has been chnaged so that it is now styled
.The page also includes monthly wage after tax and student loan repayments

jacklundy96 committed:
Added :
.Fragments have been refractored to be safer too use and tidier code via a method for each fragment
.Emplyment stats page has been chnaged so that it is now styled
.The page also includes monthly wage after tax and student loan repayments

collings22 commented:
Can't commit yet Jack, the refractored frag selector needs changing. You'll only need one try {} that'll surround the switch statement and one catch, that'll direct to the error fragment.

jacklundy96 committed:
Added :
.Fragment selector moved to one try catch rather than one per case
.changed the formatting of repayments textview

jacklundy96 commented:
Hi Dan I've made those changes and done a tweak to the repayment textview.
